partcad-cli/src/partcad_cli/click/commands/test.py

```
@@ -0,0 +1,77 @@
+def cli(ctx, package, recursive, sketch, interface, assembly, scene, object):
+    with logging.Process("Test", "this"):
+        package = package if package is not None else ""
+        if recursive:
+            start_package = ctx.get_project_abs_path(package)
+            all_packages = ctx.get_all_packages(start_package)
+            packages = list(
+                map(
+                    lambda p: p["name"],
+                    list(all_packages),
+                )
+            )
```

Ruff (0.8.2)

Unnecessary `map()` usage (rewrite using a list comprehension)

Replace `map()` with a list comprehension

(C417)